private/buf/cmd/buf/command/convert/convert.go

```diff
@@ -221,15 +229,18 @@ func run(
 		image,
 		message,
 		outputMessageRef,
+		flags.Indent,
 	)
 }
 
 // inverseEncoding returns the opposite encoding of the provided encoding,
 // which will be the default output encoding for a given payload encoding.
-func inverseEncoding(encoding bufconvert.MessageEncoding) (bufconvert.MessageEncoding, error) {
+func getDefaultEncoding(encoding bufconvert.MessageEncoding) (bufconvert.MessageEncoding, error) {
 	switch encoding {
 	case bufconvert.MessageEncodingBin:
 		return bufconvert.MessageEncodingJSON, nil
+	case bufconvert.MessageEncodingText:
```

Comment:
Adding a third option in here makes me wonder if we need to rethink the UX of this command. Why is `bin` the inverse of `text`, and vice-versa?